/**
 * Resource for testing.
 */
message PingResource {
    message GET {
        required string greeting_from_url = 1;
        required string greeting_from_query = 2;

        message Response {
            required string reply = 1;
        }
    }

    message POST {
        required string greeting = 1;

        message Response {
            required string reply = 1;
        }
    }
}
